mage_ai/frontend/pages/datasets/[slug].tsx

```diff
@@ -36,14 +36,23 @@ function Data() {
   // TODO: Move to const file 
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const metricsKeys = [
-    "count",
     "avg_null_value_count",
     "avg_invalid_value_count",
     "duplicate_row_count",
     "completeness",
     "validity",
   ];
 
+  const percentageKeys = ["completeness", "validity"];
+
+  // Map text and display priotities to backend keys.
+  const humanReadableMapping = {
+    "avg_invalid_value_count":[3, "Invalid values"],
+    "avg_null_value_count":[2, "Missing values"],
+    "completeness":[1, "Completeness"],
+    "duplicate_row_count":[4, "Duplicate values"],
+    "validity":[0, "Validity"],   
+  } || {};
```

Comment:
don't add `|| {}` when it's never going to null

Reply:
also i really don't like this. it's super confusing what you're trying to do here. it's just sorted alphabetically, can we just do a sort somewhere?

Reply:
It's actually not supposed to be sorted alphabetically. I had to change to change it so the display is sorted by what's on Figma. What's a better way to do this?

Reply:
oh oops. can you just create another array with the values in order and then create the metrics based on that array? it should keep the order if you use an array